hazelcast-tpc-engine/src/test/java/com/hazelcast/internal/tpc/AsyncServerSocketTest.java

```diff
@@ -69,14 +95,30 @@ public void reuseAddress() {
         assertFalse(socket.isReuseAddress());
     }
 
+    @Test(expected = UncheckedIOException.class)
```

Comment:
btw. I prefer `assertThrows` method from `Assert` class - you specify exactly where the exception should be thrown, so it's a little bit stronger assertion

Reply:
done